pkg/machinepool/machinepool.go

```diff
@@ -588,7 +588,7 @@ func (m *machinePool) CreateNodePools(r *rosa.Runtime, cmd *cobra.Command, clust
 		}
 	}
 
-	minReplicas, maxReplicas, replicas, autoscaling, err := manageReplicas(cmd, args, false)
+	minReplicas, maxReplicas, replicas, autoscaling, err := manageReplicas(cmd, args, false, true)
```

Comment:
NP instead of false, would it be better to supply the boolean value from the cluster instead?
cluster.Hypershift().Enabled()